utils/src/test/java/com/indix/gocd/utils/GoEnvironmentTest.java

```diff
@@ -53,4 +71,90 @@ public void shouldNotReplaceUnknownEnvVariables() {
 
         assertThat(replaced, is("COUNT:1 ${DOESNT_EXIST}"));
     }
+
+
+    @Test
```

Comment:
You're quite right: I had started with a different implementation and had some of these tests, then switched to the BooleanUtils implementation (prior to starting to use the s3 plugins, hadn't done Java for many years, so quite rusty on the capability of what's out there). I can leave out just one positive and one negative test.